### PR TITLE
Add toastr

### PR DESCRIPTION
**Pull request by ghost**
https://edu-redmine.sun-asterisk.vn/issues/31398

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -16,6 +16,7 @@ gem "config"
 gem "kaminari"
 gem "bootstrap-kaminari-views"
 gem "cocoon"
+gem 'toastr-rails'
```
Comment (manhnt-0960) [inline Gemfile]: "

**Review by manhnt-0960 — COMMENTED** [review]
phạt 5k vì lỗi ngớ ngẩn

**Comment by ghost** [comment]
ready

**Pull request merged**